radish34/__tests__/integration.test.js

```diff
@@ -271,17 +452,14 @@ describe('Buyer creates MSA, signs it, sends to supplier, supplier responds with
         .post('/graphql')
         .send({ query: postBody });
 
-      const buyerZkpPublicKey = getOrgSettings('buyer').zkpPublicKey;
-      const supplierZkpPublicKey = getOrgSettings('supplier2').zkpPublicKey;
-
       expect(res.statusCode).toEqual(200);
-      expect(res.body.data.createMSA.zkpPublicKeyOfBuyer).toEqual(buyerZkpPublicKey);
-      expect(res.body.data.createMSA.zkpPublicKeyOfSupplier).toEqual(supplierZkpPublicKey);
+      expect(res.body.data.createMSA.zkpPublicKeyOfBuyer).toEqual(buyer.zkpPublicKey);
+      expect(res.body.data.createMSA.zkpPublicKeyOfSupplier).toEqual(supplier2.zkpPublicKey);
       expect(res.body.data.createMSA.sku).toEqual('FAKE-SKU-123');
       expect(res.body.data.createMSA._id).not.toBeNull();
       expect(res.body.data.createMSA.commitments[0].commitment).toEqual(concatenateThenHash(
-        buyerZkpPublicKey,
-        supplierZkpPublicKey,
+        buyer.zkpPublicKey,
+        supplier2.zkpPublicKey,
```

Comment:
Would be good to add to the documentation/comments, about the scope/coverage of these tests, i.e. MSA for a specific org, etc.

Reply:
Do you mean something like a test coverage report produced by jest combined with more descriptive `describe()` statements for the tests?

Reply:
For example, the tests here cover sending RFP to both suppliers, but MSA is only for one supplier. So just a minor comment to clarify that these MSA tests are for Supplier 2

Reply:
Edited the test descriptions so that it is clear which parties are involved during each set of tests